scripts/lint-style.sh

```diff
@@ -38,7 +38,7 @@ set -exo pipefail
 touch scripts/style-exceptions.txt
 
 # Assuming `archive/` and `counterexamples/` are ported too, they need to be added here.
-find Mathlib -name '*.lean' | xargs ./scripts/lint-style.py
+git ls-files '*.lean' | grep -v '^test/' | xargs ./scripts/lint-style.py
```

Comment:
```suggestion
find Mathlib Counterexamples Archive -name '*.lean' | xargs ./scripts/lint-style.py
```

Reply:
IMHO, `git ls-files` is better for people like me who want to run the script locally and may have files not added to `git`.